i18n/he.yaml

```diff
@@ -1,21 +1,21 @@
 - id: prev_page
-  translation: "קודם"
+  translation: "הקודם"
 
 - id: next_page
   translation: "הבא"
 
 - id: read_time
   translation:
     one : "דקה אחת"
-    other: "דקות {{ .Count }}"
+    other: "{{ .Count }} דקות"
 
 - id: words
   translation:
-    one : "word"
-    other: "מילים {{ .Count }} "
+    one : "מילה"
```

Comment:
I suggest

> מילה אחת

This phrasing is also consistent with `read_time.one`

Reply:
Done ✅